### DISCUSSION TITLE
Overriding Default Date-Time Format Behavior in date-filter Component

### DISCUSSION THREAD

**Discussion by jbumaniss** [laravel/nova-issues#5878] [Ideas & Feature Requests]
Within Laravel Nova, I've noticed that the native 'date-filter' component's date-time format automatically conforms to the browser's display language setting. The specific area of contention arises between the formats predominant in the USA and Europe:

- When the browser's language is set to English US (en-US), the format appears as: 'mm/dd/yyyy'.

- Conversely, for settings like English GB (en-GB), it displays as:  'dd.mm.yyyy'.

While this adaptability to the browser's language can be beneficial in many scenarios, there are instances, including in my application, where a consistent date-time format is desirable regardless of these regional variations.

#### Static Date-Time Format Setting in date-filter:

Introduce the capability for developers to specify and lock in a particular date-time format for the 'date-filter' component. This format should remain consistent and uninfluenced by the browser's display language setting. Such an addition will cater to applications that aim for a standard display format across various regional settings.

### Rationale:
Adjusting to browser language settings might be standard behavior, but it's imperative to acknowledge the practical needs where a consistent date-time display is beneficial. Whether catering to international audiences or ensuring uniform data representation, providing this degree of customization aligns with the adaptive and user-centric nature of Laravel Nova.

### Steps to Reproduce the Current Behavior:

1. Deploy a fresh Laravel Nova instance.

2. Incorporate the native date-filter component within a Nova filter.
3. Switch between browser's display languages, specifically between English US (en-US) and English GB (en-GB).
4. Observe the shifting date-time format within the 'date-filter' based on these language preferences.

With this feature, developers can seamlessly cater to both international and region-specific audiences, ensuring a consistent and predictable user experience within Laravel Nova.

**Comment by ghost** [comment]
Hi @crynobone. Any chance this could be implemented?

**Reply by crynobone** [reply]
We'll look into this if HTML spec support such customisation for `<input type="datetime-local" />`

**Reply by ghost** [reply]
Hi @crynobone. Any news regarding this one? Thanks!

**Reply by crynobone** [reply]
No plan for this at the moment.
